docs/patch.md

```diff
@@ -123,6 +123,39 @@ Here's a minimal example showing how to clear a given path:
 patch.clear('custom.pets');
 ```
 
+### remove
+
+This method removes all values that match in a collection.
```

Comment:
Since the `add` method has the description `This method adds a value to a collection.`, what do you think about writing it like this?

`This method removes values from a collection.`

Since it removes all occurrences equal to the given value, I would use the plural form.

Reply:
Makes sense, thank you!